iina/AppDelegate.swift

```diff
@@ -201,6 +201,9 @@ class AppDelegate: NSObject, NSApplicationDelegate {
       }
     }
 
+    // Hide Window > "Enter Full Screen" menu item, becuase this is already present in the Video menu
+    UserDefaults.standard.set(false, forKey: "NSFullScreenMenuItemEverywhere")
```

Comment:
This is the wrong place to set this. That is what is causing this:
> Although it's set when IINA starts, it will not take effect until the app is closed & reopened.

The only Apple documentation on this seems to be [AppKit Release Notes (macOS 10.12 and Earlier)](https://developer.apple.com/library/archive/releasenotes/AppKit/RN-AppKitOlderNotes/index.html#10_11FullScreen) which says:
>AppKit automatically creates an "Enter Full Screen" menu item after the application finishes launching if an equivalent menu item isn't found. If this menu item should not be created for your app, before NSApplicationDidFinishLaunchingNotification is sent you may set the NSFullScreenMenuItemEverywhere default to NO.

This statement has been added to the method `applicationDidFinishLaunching` which is after the `NSApplicationDidFinishLaunchingNotification` notification. I tested moving this statement into the `applicationWillFinishLaunching` method right after the "App will launch" log is emitted. I used `defaults` to make sure that key was not set. The menu item was hidden on the first run. I did not have to restart IINA for it to take effect.